### ISSUE TITLE
storr does not install (R6 update clash?)

### ISSUE THREAD

**Issue by RemkoDuursma** [richfitz/storr#8]
With `devtools::install_github(c("richfitz/storr"))`,  I get

```
* installing *source* package 'storr' ...
** R
** inst
** tests
** preparing package for lazy loading
R6Class driver_external: 'lock' argument has been renamed to 'lock_objects' as of version 2.1.This code will continue to work, but the 'lock' option will be removed in a later version of R6
Error in R6::R6Class("storr_mangled", public = storr_mangled_methods()) : 
  Cannot add a member with reserved name 'clone'.
Error : unable to load R code in package 'storr'
ERROR: lazy loading failed for package 'storr'
```

**Comment by richfitz** [comment]
Thanks.  There was a big update to R6 the other day and it looks like that's broken something with storr.  Will look into this.

**Comment by richfitz** [comment]
OK, should be fixed as of richfitz/storr@f95ee2e

**Comment by RemkoDuursma** [comment]
yep - thanks
